### ISSUE TITLE
azir-Slider does not update after the first render

### ISSUE THREAD

**Issue by mynameisguy** [abodehq/azir#1]
When using the example for the azir-slider:
```
import Slider from "azir-slider";
---
<Slider
  value={30}
  step={10}
  minimumValue={0}
  maximumValue={100}
  onStart={value => console.log(value)}
  onChange={value => console.log(value)}
  onComplete={value => console.log(value)}
  trackColor="red"
  thumbColor="blue"
  progressTrackColor="green"
  thumbSize={50}
  trackSize={30}
/>
```

it works like in the example.
when changing the value field to `this.state.position` the slider is stuck at the first value given.
logging the `this.state.position` in the render function shows it gets updated.

**Comment by abodehq** [comment]
I have test it and it works fine , you need to set the state when you drage the slide using the onChange method please chceck below example :
  const [value,setValue] = useState(30);
<Slider
  value={value}
  step={10}
  minimumValue={0}
  maximumValue={100}
  onStart={value => setValue(value)}
  onChange={value => setValue(value)}
  onComplete={value => setValue(value)}
  trackColor="red"
  thumbColor="blue"
  progressTrackColor="green"
  thumbSize={50}
  trackSize={30}
/>

hope it works now .

**Comment by GManzato** [comment]
I'm having the same problem, if I update the value without using onChange , it's not updated, I have a fix , just need to add this in the useEffect.
```
setValue(new Animated.Value(_value));

```

**Comment by vikjovanov** [comment]
Hi,

I have same issue... when I render azirSlider dynamically my thumb stay at the default value of 1 but doesn't re-render if I change my value without using onChange()

**Comment by abodehq** [comment]
vikjovanov can you please post a working example so i can figure out the problem ??

**Comment by abodehq** [comment]
Fixed please install azir-slider V1.0.1

**Comment by mynameisguy** [comment]
works. thanks
